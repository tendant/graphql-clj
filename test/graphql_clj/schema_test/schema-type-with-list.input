type TypeName {
  optListOfOpt:[String]
  reqListOfOpt:[String]!
  optListOfReq:[String!]
  reqListOfReq:[String!]!
}
