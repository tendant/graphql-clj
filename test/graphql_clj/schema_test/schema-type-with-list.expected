{:objects
 {:TypeName
  {:fields
   {:optListOfOpt {:type (list String)},
    :reqListOfOpt {:type (non-null (list String))},
    :optListOfReq {:type (list (non-null String))},
    :reqListOfReq {:type (non-null (list (non-null String)))}}}}}
